examples/example_notebooks/stable_baselines3_example.ipynb

```diff
@@ -0,0 +1,456 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, before you start your evaluation you have to load a trained agent or take the trained agent from above which is still saved in the variable model. To load a trained agent you simply have to call the .load() function of your algorithm with the respective path. If you have executed the provided code above you can uncomment the first line of code and comment the second one."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#model = DQN.load(Path(__file__).parent / \"saved_agents\" / \"TutorialAgent\")  #your agent\n",
+    "model = DQN.load(Path.cwd() / \"saved_agents\" / \"TutorialPreTrainedAgent\") #pre-trained agent"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.2 Taking a look at the mean reward per episode during training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The RewardLogger callback saved the mean reward per episode during training. There you can observe, if the training reward did grow over time. Uncomment the first line and comment the second line if you have executed the training cell. An exemplary graph could look like this:\n",
+    "\n",
+    "![TrainingRew](img/TrainingRewEpisode.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#rewards = np.load(Path.cwd() / \"saved_agents\" / \"EpisodeRewards.npy\")[1:] # your training rewards\n",
+    "rewards = np.load(Path.cwd() / \"saved_agents\" / \"PreTrainedEpisodeRewards.npy\")[1:] # pre-trained training rewards\n",
+    "plt.grid(True)\n",
+    "plt.xlim(0,len(rewards))\n",
+    "plt.ylim(min(rewards), 0)\n",
+    "plt.yticks(np.arange(min(rewards), 1, 1.0))\n",
+    "plt.tick_params(axis = 'y', left = False, labelleft = False)\n",
+    "plt.xticks(np.arange(0, len(rewards), 10))\n",
+    "plt.xlabel('#Episode')\n",
+    "plt.ylabel('Mean Reward per Episode (Qualitative)')\n",
+    "plt.plot(rewards)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.3 Taking a look at a test trajectory"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can to take a look at a test trajectory to see how well your trained agent is able to control the currents to follow the test trajectory. For the agent to decide for an action given an observation you can just call its .predict() function. The key deterministic is important so that the agent is not using a stochastic policy like epsilon greedy but is instead chosing an action greedily. You can see a possible trajectory below. The blue trajectory is the one induced by the agent's actions while the green one is optimal:\n",
+    "\n",
+    "![Motor Setup](img/example_trajectory2.png)"
```

Comment:
again a static picture of some visualization that should be generated by the notebook.
Please don't do this.